### PR TITLE
Fix #7148 Add readthedocs.yml and upgrade projex files

### LINKED ISSUE #7148: readthedocs & projex upgrade

### PR DESCRIPTION
**Pull request by robnagler**

### CODE REVIEW

File: sirepo/sirepo_console.py [hunk 1]
```diff
@@ -1,21 +1,17 @@
-# -*- coding: utf-8 -*-
-"""SiRepo front-end command line for :mod:`pykern.pkcli`.
+"""Front-end command line for :mod:`sirepo`.
 
-Example:
+See :mod:`pykern.pkcli` for how this module is used.
 
-:copyright: Copyright (c) 2015 RadiaSoft LLC.  All Rights Reserved.
-:license: http://www.apache.org/licenses/LICENSE-2.0.html
+:copyright: Copyright (c) 2015-2024 RadiaSoft LLC..  All Rights Reserved.
```
Comment (rorour) [inline sirepo/sirepo_console.py:5]: superfluous dot

File: README.md [hunk 2]
```diff
@@ -29,7 +29,7 @@ $ curl https://sirepo.run | bash
 
 For this to work, you will need to [install the prerequisites](https://github.com/radiasoft/download/blob/master/README.md#requirements).
 
-[API Documentation is available on Read the Docs.](http://sirepo.readthedocs.org)
+[API Documentation is available on Read the Docs.](https://sirepo.readthedocs.io)
 
 ### Manual Install with Docker
```
Comment (rorour) [inline README.md:35]: Might as well update the copyright year on line 52

**Review by rorour — APPROVED** [review]
nits

**Comment by robnagler** [comment]
This PR depends on pykern prod release.

File: README.md (hunk 2 above)
Comment (robnagler) [reply README.md:35]: 2ad63f9b5f60e50c0cd8d0b67e643a3e756bda8e

File: sirepo/sirepo_console.py (hunk 1 above)
Comment (robnagler) [reply sirepo/sirepo_console.py]: 2ad63f9b5f60e50c0cd8d0b67e643a3e756bda8e

**Pull request merged**